plugins/multiplexer/ats-multiplexer.cc

```diff
@@ -39,6 +41,7 @@
 const size_t DEFAULT_TIMEOUT = 1000000000000;
 
 Statistics statistics;
+bool skip_post_put = false;
```

Comment:
Instead of a variable here. Should we keep all the parameters in one place under the Instance object? 
Perhaps we can add another variable to the Instance object for this flag insead?

Reply:
@shukitchan will do